tasks.py

```diff
@@ -0,0 +1,261 @@
+import git
+import json
+import os
+import sys
+
+from shutil import copyfile
+from invoke import task
+
+
+APP_NAME = "fecfile-web-app"
+ORG_NAME = "fec-fecfileonline-prototyping"
+APP_BUILD_NAME = "fec-eFilling"
```

Comment:
Do we need `APP_BUILD_NAME`? If so, should it be "fec-eFiling"?

Reply:
Doesn't look like it is needed.  It was present in the original tasks.py.  I'm removing it.